src/mctpd.c

```diff
@@ -3512,6 +3901,41 @@ static int add_net(ctx *ctx, int net)
 	return 0;
 }
 
+static int add_interface(ctx *ctx, int ifindex)
+{
+	link_userdata *lmUserData = (link_userdata *)malloc(sizeof(link_userdata));
+	int rc;
+
+	int net = mctp_nl_net_byindex(ctx->nl, ifindex);
+	if (net <= 0) {
+		warnx("Can't find link index %d\n", ifindex);
```

Comment:
No newline needed on `warn` / `warnx` / `err` / `errx`.

Reply:
you've resolved this comment, but there are still a bunch of new "\n" in warnx here.